src/pages/Detail/Booking/Booking.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { MyDatePicker } from '../../../styles/Content.styled';
+import { CalendarContainer } from 'react-datepicker';
+import moment from 'moment';
+import { ko } from 'date-fns/esm/locale';
+import { BookingButton, CountButton } from '../../../styles/Button.styled';
+import {
+  BookingForm,
+  DateForm,
+  CountForm,
+  CheckForm,
+  CenterForm,
+} from '../../../styles/Form.styled';
+import {
+  BetweenDiv,
+  Underline,
+  H1,
+  H2,
+  UnderlinedH2,
+  BoldH2,
+  H3,
+  BoldH3,
+  Span,
+  BigSpan,
+} from '../../../styles/Content.styled';
+import { CalendarFlexContainer } from '../../../styles/Container.styled';
+import './DatePicker.css';
+
+const Booking = ({ productData }) => {
+  const [count, setCount] = useState(0);
+  const [dateRange, setDateRange] = useState([null, null]);
+  const params = useParams();
+  const [startDate, endDate] = dateRange;
+  const checkInDate = startDate && moment(startDate).format('YYYY-MM-DD');
+  const checkOutDate = endDate && moment(endDate).format('YYYY-MM-DD');
+  const duration = moment
+    .duration(moment(checkOutDate).diff(moment(checkInDate)))
+    .asDays();
+  const wonPrice =
+    productData.length &&
+    productData[params.id - 1].price
+      .toString()
+      .replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  const sumPrice =
```

Comment:
```suggestion
  const [count, setCount] = useState(0);
  const [dateRange, setDateRange] = useState([null, null]);
  const [startDate, endDate] = dateRange;

  const params = useParams();
  const checkInDate = startDate && moment(startDate).format('YYYY-MM-DD');
  const checkOutDate = endDate && moment(endDate).format('YYYY-MM-DD');
  
  const duration = moment
    .duration(moment(checkOutDate).diff(moment(checkInDate)))
    .asDays();
    
  const wonPrice =
    productData.length &&
    productData[params.id - 1].price
      .toString()
      .replace(/\B(?=(\d{3})+(?!\d))/g, ',');
      
  const sumPrice =
```

코드의 가독성을 위해 이런 식으로 코드 간격을 띄워주세요